Technical-Steering-Committee/squads.md

```diff
@@ -0,0 +1,108 @@
+# Squads
+
+The squad is a group of people contributing and participating in the Zowe project. The group owns either some piece of functionality or one or more modules. Every squad needs to have representative in TSC and paticipate in relevant working groups. 
+
+## Maturity
+
+This chapter outlines what levels of maturity are to be found within the Zowe project and the ways to transition between the different stages within the lifecycle.
+
+## Squads
+
+- **Proposal** - Idea presented to the TSC and ZAC based on the process outlined in [Proposal Process](#Proposal Acceptance Process)
```

Comment:
The link isn't appearing correctly for me. I think it should be `[Proposal Process](#proposal-acceptance-process)`